pkg/metric/scrape_memory.go

```diff
@@ -110,6 +115,16 @@ func scrapeStarterValues(ch chan<- prometheus.Metric) error {
 	ch <- prometheus.MustNewConstMetric(
 		memoryRunnerQueueConcurrencyDeleting, prometheus.GaugeValue, float64(countRunnerDeletingNow), labelRunner)
 
+	deleteRetried := 0
+	runner.DeleteRetryCount.Range(func(key, value any) bool {
+		deleteRetried += value.(int)
+		return true
+	})
+
+	createdRunners := runner.CreatedRunners.Load()
+	ch <- prometheus.MustNewConstMetric(
+		memoryRunnerDeleteRetriedRatio, prometheus.GaugeValue, float64(createdRunners)/float64(deleteRetried), labelRunner)
```

Comment:
Please expose two metrics ( `runner_delete_retried` and `runner_delete_try` ). The metrics should not calculate in-app, We should calculate in frontend (e.g. Prometheus, Grafana)